test/CMakeLists.txt

```diff
@@ -338,7 +338,7 @@ function(add_custom_test NAME)
     set(HALF_TEST_DEFAULT FALSE)
     option_support_check(${PARSE_HALF_ENABLED} ${PARSE_HALF_DISABLED} ${HALF_TEST_DEFAULT} is_half_check)
     bool_and_f(${MIOPEN_TEST_HALF} ${is_half_check} is_half_check)
-
+    
```

Comment:
:warning: Unwanted whitespace changes!
I will revert them myself to speed up the process.

Reply:
> warning Unwanted whitespace changes!
> I will revert them myself to speed up the process.

Thank you.   Conflicts is not solved while reverting

Reply:
You are welcome :smoking: 
I would like to push it through CI and merge ASAP.